sdcm/provision/helpers/certificate.py

```diff
@@ -46,3 +67,187 @@ def install_encryption_at_rest_files(remoter):
+        .not_valid_before(datetime.utcnow())
+        .not_valid_after(datetime.utcnow() + timedelta(days=valid_days))
+        .add_extension(x509.SubjectAlternativeName(alt_names), critical=False)
+        .add_extension(x509.BasicConstraints(ca=False, path_length=None), critical=True)
+        .add_extension(x509.SubjectKeyIdentifier.from_public_key(private_key.public_key()), critical=False)
+        .sign(sign_key, hashes.SHA256())
+    )
+
+    with open(file=cert_file, mode='wb') as file:
+        file.write(cert.public_bytes(serialization.Encoding.PEM))
+    with open(file=key_file, mode='wb') as file:
+        file.write(
+            private_key.private_bytes(
+                encoding=serialization.Encoding.PEM,
+                format=serialization.PrivateFormat.PKCS8,
+                encryption_algorithm=serialization.NoEncryption())
+        )
+    if keystore_file:
+        with open(keystore_file, 'ab') as file:
+            file.write(
+                private_key.private_bytes(
+                    encoding=serialization.Encoding.PEM,
+                    format=serialization.PrivateFormat.PKCS8,
+                    encryption_algorithm=serialization.NoEncryption())
+            )
+            file.write(cert.public_bytes(serialization.Encoding.PEM))
+
+
+def convert_pem_cert_to_jks(
+        cert_file: str, cert_key_file: str, pkcs12_file: str, jks_file: str,
+        cert_key_password: str = 'scylladb', jks_password: str = 'cassandra') -> None:
+    """Convert a PEM formatted certificate to a Java keystore."""
+    with open(cert_file, 'rb') as file:
+        cert = x509.load_pem_x509_certificate(file.read())
+    with open(cert_key_file, 'rb') as file:
+        key = serialization.load_pem_private_key(file.read(), password=cert_key_password.encode())
+
+    # Serialize PKCS12 keystore container and write it to a file
+    pkcs12 = serialize_key_and_certificates(
+        name=b'scylladb', key=key, cert=cert, cas=None,
+        encryption_algorithm=serialization.BestAvailableEncryption(jks_password.encode()))
+    with open(pkcs12_file, 'wb') as file:
+        file.write(pkcs12)
+
+    # Convert a PKCS12 formatted keystore to a JKS keystore
+    cmd = (
+        f'keytool -importkeystore -noprompt '
+        f'-srckeystore {pkcs12_file} -srcstoretype PKCS12 -srcstorepass {jks_password} '
+        f'-destkeystore {jks_file} -deststoretype JKS -deststorepass {jks_password} -destkeypass {jks_password}')
+    subprocess.run(cmd, shell=True, check=True)
+
+
+def cleanup_ssl_config():
+    """Remove SCT ssl_config files that are dynamically created during the test"""
+    for file in [CA_CERT_FILE, CA_KEY_FILE, SERVER_CSR_FILE, CLIENT_CERTFILE, CLIENT_KEYFILE, CLIENT_TRUSTSTORE,
+                 CLIENT_KEYSTORE_FILE, SERVER_CERT_FILE, SERVER_KEY_FILE, SERVER_KEYSTORE_FILE, DER_CERT_FILE,
+                 JKS_TRUSTSTORE_FILE, PKCS12_KEYSTORE_FILE]:
+        if os.path.exists(file):
```

Comment:
use `Path.unlink(missing_ok=False)`

also maybe save those as Path object to begin with ?